Submitted By: Andrew Benton (b3nton@gmail.com)
Date: 2011-20-12
Initial Package Version: 1.42
Origin: Andrew Benton
Description: Fixes xsnow for a modern system

--- xsnow-1.42/Makefile~	2001-12-15 23:43:31.000000000 +0000
+++ xsnow-1.42/Makefile	2011-12-20 16:27:44.711487956 +0000
@@ -315,7 +315,7 @@
 # X Window System make variables; these need to be coordinated with rules
 
              XTOP = $(TOP)
-           BINDIR = /usr/X11R6/bin
+           BINDIR = /usr/bin
      BUILDINCROOT = $(TOP)/exports
       BUILDINCDIR = $(BUILDINCROOT)/include
       BUILDINCTOP = ../..
@@ -872,46 +872,7 @@
 # dependencies generated by makedepend
 
 # DO NOT DELETE
-xsnow.o: xsnow.c /usr/X11R6/include/X11/Intrinsic.h \
- /usr/X11R6/include/X11/Xlib.h /usr/include/sys/types.h \
- /usr/include/features.h /usr/include/sys/cdefs.h \
- /usr/include/gnu/stubs.h /usr/include/bits/types.h \
- /usr/lib/gcc-lib/i486-suse-linux/2.95.2/include/stddef.h \
- /usr/include/time.h /usr/include/endian.h /usr/include/bits/endian.h \
- /usr/include/sys/select.h /usr/include/bits/select.h \
- /usr/include/bits/sigset.h /usr/include/sys/sysmacros.h \
- /usr/X11R6/include/X11/X.h /usr/X11R6/include/X11/Xfuncproto.h \
- /usr/X11R6/include/X11/Xosdefs.h /usr/X11R6/include/X11/Xutil.h \
- /usr/X11R6/include/X11/Xresource.h /usr/include/string.h \
- /usr/X11R6/include/X11/Core.h /usr/X11R6/include/X11/Composite.h \
- /usr/X11R6/include/X11/Constraint.h /usr/X11R6/include/X11/Object.h \
- /usr/X11R6/include/X11/RectObj.h /usr/X11R6/include/X11/Xos.h \
- /usr/include/fcntl.h /usr/include/bits/fcntl.h \
- /usr/include/sys/stat.h /usr/include/bits/stat.h \
- /usr/include/unistd.h /usr/include/bits/posix_opt.h \
- /usr/include/bits/confname.h /usr/include/getopt.h \
- /usr/include/sys/time.h /usr/include/bits/time.h \
- /usr/X11R6/include/X11/Xarch.h /usr/X11R6/include/X11/xpm.h \
- /usr/include/stdio.h \
- /usr/lib/gcc-lib/i486-suse-linux/2.95.2/include/stdarg.h \
- /usr/include/libio.h /usr/include/_G_config.h /usr/include/wchar.h \
- /usr/include/bits/wchar.h /usr/include/gconv.h \
- /usr/include/bits/stdio_lim.h /usr/include/math.h \
- /usr/include/bits/huge_val.h /usr/include/bits/mathdef.h \
- /usr/include/bits/mathcalls.h \
- /usr/lib/gcc-lib/i486-suse-linux/2.95.2/include/float.h \
- /usr/include/signal.h /usr/include/bits/signum.h \
- /usr/include/bits/siginfo.h /usr/include/bits/wordsize.h \
- /usr/include/bits/sigaction.h /usr/include/bits/sigcontext.h \
- /usr/include/asm/sigcontext.h /usr/include/bits/sigstack.h \
- /usr/include/ucontext.h /usr/include/sys/ucontext.h \
- /usr/lib/gcc-lib/i486-suse-linux/2.95.2/include/limits.h \
- /usr/lib/gcc-lib/i486-suse-linux/2.95.2/include/syslimits.h \
- /usr/include/limits.h /usr/include/bits/posix1_lim.h \
- /usr/include/bits/local_lim.h /usr/include/linux/limits.h \
- /usr/include/bits/posix2_lim.h /usr/include/bits/xopen_lim.h \
- /usr/include/stdlib.h /usr/include/bits/waitflags.h \
- /usr/include/bits/waitstatus.h /usr/include/alloca.h xsnow.h \
+xsnow.o: xsnow.c xsnow.h \
  pixmaps/snow00.xbm pixmaps/snow01.xbm pixmaps/snow02.xbm \
  pixmaps/snow03.xbm pixmaps/snow04.xbm pixmaps/snow05.xbm \
  pixmaps/snow06.xbm pixmaps/tannenbaum.xbm pixmaps/BigSanta1.XPM \
@@ -926,32 +887,5 @@
  pixmaps/RegularSanta3.XPM pixmaps/RegularSanta4.XPM \
  pixmaps/RegularSantaRudolf1.XPM pixmaps/RegularSantaRudolf2.XPM \
  pixmaps/RegularSantaRudolf3.XPM pixmaps/RegularSantaRudolf4.XPM \
- toon.h /usr/X11R6/include/X11/extensions/shape.h
-toon_root.o: toon_root.c toon.h /usr/X11R6/include/X11/Intrinsic.h \
- /usr/X11R6/include/X11/Xlib.h /usr/include/sys/types.h \
- /usr/include/features.h /usr/include/sys/cdefs.h \
- /usr/include/gnu/stubs.h /usr/include/bits/types.h \
- /usr/lib/gcc-lib/i486-suse-linux/2.95.2/include/stddef.h \
- /usr/include/time.h /usr/include/endian.h /usr/include/bits/endian.h \
- /usr/include/sys/select.h /usr/include/bits/select.h \
- /usr/include/bits/sigset.h /usr/include/sys/sysmacros.h \
- /usr/X11R6/include/X11/X.h /usr/X11R6/include/X11/Xfuncproto.h \
- /usr/X11R6/include/X11/Xosdefs.h /usr/X11R6/include/X11/Xutil.h \
- /usr/X11R6/include/X11/Xresource.h /usr/include/string.h \
- /usr/X11R6/include/X11/Core.h /usr/X11R6/include/X11/Composite.h \
- /usr/X11R6/include/X11/Constraint.h /usr/X11R6/include/X11/Object.h \
- /usr/X11R6/include/X11/RectObj.h /usr/X11R6/include/X11/Xos.h \
- /usr/include/fcntl.h /usr/include/bits/fcntl.h \
- /usr/include/sys/stat.h /usr/include/bits/stat.h \
- /usr/include/unistd.h /usr/include/bits/posix_opt.h \
- /usr/include/bits/confname.h /usr/include/getopt.h \
- /usr/include/sys/time.h /usr/include/bits/time.h \
- /usr/X11R6/include/X11/Xarch.h /usr/X11R6/include/X11/xpm.h \
- /usr/X11R6/include/X11/extensions/shape.h \
- /usr/X11R6/include/X11/Xatom.h /usr/X11R6/include/X11/Xproto.h \
- /usr/X11R6/include/X11/Xmd.h /usr/X11R6/include/X11/Xprotostr.h \
- /usr/include/stdio.h \
- /usr/lib/gcc-lib/i486-suse-linux/2.95.2/include/stdarg.h \
- /usr/include/libio.h /usr/include/_G_config.h /usr/include/wchar.h \
- /usr/include/bits/wchar.h /usr/include/gconv.h \
- /usr/include/bits/stdio_lim.h
+ toon.h
+toon_root.o: toon_root.c toon.h
